pkg/admission/admission.go

```diff
@@ -154,6 +154,23 @@ func (a *ParallelAdmission) ValidateResources(ctx context.Context, localResource
 			Name:      objName,
 		}
 
+		// If the object includes a pod spec (i.e. Deployment), create a Pod object out of it for validation.
+		// For admission only pod resources will be enforced, deployments won't.
```

Comment:
Could you formulate this a bit like `"The admission only returns warnings for pod controllers, the enforcement happens for direct pod resources only"` (if the part about the warnings is true).